### PR TITLE
Github repo summary

### PR DESCRIPTION
**Pull request by krzyurb**
Basic repository summary command. It shows data about Average time PR are open, average description length, number of open issues and latest contributors list

**Usage example**
`github repo bob-the-bot`

**JIRA**
TECH-391

### CODE REVIEW

File: src/github/src/github.messages.js [hunk 1]
```diff
@@ -5,6 +5,43 @@ const { colors } = require('../config/config.json');
 
 const locales = new Locales({ plugin: 'github' });
 
+const repoSummaryMessage = (summary) => {
+  const builder = new MessageBuilder();
+
+  if (!summary.pullRequests.length) {
+    builder.addAttachment({
+      text: locales.get('repo-summary-no-data'),
```
Comment (Getriax) [inline src/github/src/github.messages.js]: What do you think about checking if the pull requests are available [here](https://github.com/netguru/bob-the-bot/pull/133/files#diff-fe89a02a1d34f16860f4ee0df264227bR65)? Throwing an error there would make it possible to be caught by `RespondMultiplexer` and no further code will be executed 🙂

Comment (krzyurb) [reply src/github/src/github.messages.js]: done

**Comment by krzyurb** [comment]
@Getriax fix done

**Comment by krzyurb** [comment]
@Getriax done

**Pull request merged**